ecs-cli/modules/cli/local/ps_app.go

```diff
@@ -63,18 +68,39 @@ const (
 // If the --all flag is provided, then list all local ECS task containers.
 // If the --json flag is provided, then output the format as JSON instead.
 func Ps(c *cli.Context) {
+	if err := validateOptions(c); err != nil {
+		logrus.Fatalf(err.Error())
+	}
 	containers := listContainers(c)
 	displayContainers(c, containers)
 }
 
+func validateOptions(c *cli.Context) error {
```

Comment:
nit: can you move this fn to `create_app.go`? It seems like it should belong there.

Reply:
fixed